build.gradle

```diff
@@ -121,6 +121,7 @@ def createSpotlessTarget = { pattern ->
             'secrets',
             'charts', // Helm charts often have injected template strings that will fail general linting. Helm linting is done separately.
             'resources/seed/*_specs.yaml', // Do not remove - this is necessary to prevent diffs in our github workflows, as the file diff check runs between the Format step and the Build step, the latter of which generates the file.
+            'resources/seed/*_catalog.json', // Do not remove - this is also necessary to prevent diffs in our github workflows
```

Comment:
Need to ignore this from the `format` to avoid failures when CI checks if everything has been formatted. Same has already been happening above for the generated spec yamls.